homeassistant/components/number/device_action.py

```diff
@@ -0,0 +1,80 @@
+"""Provides device actions for Number."""
+from typing import Any, Dict, List, Optional
+
+import voluptuous as vol
+
+from homeassistant.const import (
+    ATTR_ENTITY_ID,
+    CONF_DEVICE_ID,
+    CONF_DOMAIN,
+    CONF_ENTITY_ID,
+    CONF_TYPE,
+)
+from homeassistant.core import Context, HomeAssistant
+from homeassistant.helpers import entity_registry
+import homeassistant.helpers.config_validation as cv
+
+from . import DOMAIN, const
+
+ACTION_SCHEMA = cv.DEVICE_ACTION_BASE_SCHEMA.extend(
+    {
+        vol.Required(CONF_TYPE): "set_value",
```

Comment:
I think since `"set_value"`  is used in a few places, then it should be a constant

Reply:
Thanks, done